freeipa/src/main/java/com/sequenceiq/freeipa/entity/Stack.java

```diff
@@ -147,6 +148,10 @@ public class Stack implements AccountAwareResource, OrchestratorAware {
     @SecretValue
     private Secret ccmParameters = Secret.EMPTY;
 
+    @ElementCollection(fetch = FetchType.EAGER)
+    @Column(name = "recipe")
+    private Set<String> recipes;
```

Comment:
similarly to env entity, I'm not sure this is a good idea to have an extra join every time we fetch the stack entity. we do that a lot when creating a freeipa client, and we don't need recipes in that case